models/store/src/store/store-draft/presets/sample-data-goodstore/the-good-store.ts

```diff
@@ -0,0 +1,11 @@
+import { LocalizedString } from '@commercetools-test-data/commons';
+import type { TStoreDraftBuilder } from '../../../types';
+import * as StoreDraft from '../../index';
+
+const the_good_store = (): TStoreDraftBuilder =>
+  StoreDraft.presets
+    .empty()
+    .key('the-good-store')
+    .name(LocalizedString.presets.empty()['en-US']('The Good Store'))
+    .name(LocalizedString.presets.empty()['en-GB']('The Good Store'));
```

Comment:
Line 10 here will overwrite line 9; do we want both? We could do something like 
`.name(LocalizedString.presets.empty()['en-US']('The Good Store')['en-GB']('The Good Store'))`
to set the name for more than one language if that's what we need. Is [this](https://github.com/frontastic-developers/customer-thegoodstore/blob/master/getting-started/data/stores.json) what we want the end result to be?

Reply:
Made the update. Thanks for highlighting